ptero_petri/api/wsgi.py

```diff
@@ -9,11 +10,13 @@
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--port', type=int, default=5000)
     parser.add_argument('--debug', action='store_true', default=False)
     return parser.parse_args()
 
 
 if __name__ == '__main__':
     args = parse_args()
-    app.run(port=args.port, debug=args.debug)
+    app.run(
+        host=os.environ.get('PTERO_PETRI_HOST'),
+        port=os.environ.get('PTERO_PETRI_PORT'),
```

Comment:
Without a default specified `.get` will return `None`.  I guess that's just as good as using the square-bracket notation and getting a KeyError, unless `app.run` has defaults for host and port.  In that case we should definately switch to square-bracket... these env variables are not optional.